arch/RISC-V/SiFive/drivers/fe310.ads

```diff
@@ -30,10 +32,57 @@
 ------------------------------------------------------------------------------
 
 with HAL; use HAL;
+with FE310_SVD.PRIC;
 
 package FE310 is
 
+   LF_Clock_Frequency : constant := 32768;
+
    function CPU_Frequency return UInt32;
    --  Compute CPU frequency
+
+   procedure Load_Internal_Oscilator_Calibration;
+   --  Read the calibration setting from the OTP memory ant write it to the
+   --  oscillator configuration register.
+   --  After execution of this procedure, the (undivided) internal oscillator
+   --  frequency should be about 72 MHz
+
+   subtype PLL_Output_Divider is Integer range 1 .. 128;
+
+   procedure Use_Crystal_Oscillator (Divider : in PLL_Output_Divider := 1)
```

Comment:
We omit the `in` keyword when a subprogram argument is input only. You have other occurrence of this in the package.